funnel/templates/update_details.html.jinja2

```diff
@@ -1,6 +1,6 @@
 {% set current_page = 'update' %}
```

Comment:
For a future PR: these `set` tags have to be removed, and the template should use `current_view.<whatever>`. I will refactor the views to reflect the page layouts and sections.

Reply:
Added an issue for this